lib/models/ReplicationConfiguration.js

```diff
@@ -357,7 +367,11 @@ class ReplicationConfiguration {
      * @return {undefined}
      */
     parseConfiguration() {
-        const err = this._parseRole() || this._parseRules();
+        let err = this._parseRules();
+        if (err) {
+            return err;
+        }
+        err = this._parseRole();
```

Comment:
You can `return this._parseRole();` for brevity and remove the if clause below it.

Reply:
True! Updated.